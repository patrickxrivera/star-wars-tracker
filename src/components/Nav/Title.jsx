import React from 'react';
import styled from 'styled-components';

const Container = styled.div`
  display: flex;
  flex-direction: column;
  border: 2px solid red;
  text-align: center;
`;

const Header = styled.div`
  font-size: 1.25rem;
  font-weight: bold;
`;

const Subscript = styled.div`
  font-size: 1.25rem;
  margin-top: 0.3rem;
  color: hsl(255, 50%, 85%);
`;

function Title() {
  return (
    <Container>
      <Header>Star Wars Tracker</Header>
      <Subscript>The API Strikes Back</Subscript>
    </Container>
  );
}

export default Title;
