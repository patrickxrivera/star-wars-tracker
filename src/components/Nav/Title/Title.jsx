import React from 'react';

import Header from './TitleStyles.jsx';

const Title = () => (
  <div>
    <Header>The API Strikes Back</Header>
  </div>
);

export default Title;
