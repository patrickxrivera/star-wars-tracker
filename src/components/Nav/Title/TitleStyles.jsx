import styled from 'styled-components';

const Header = styled.div`
  font-size: 1.25rem;
  font-weight: bold;
`;

const Link = styled.a`
  text-decoration: none;
  color: #52565f;

  &:visited {
    color: default;
  }
`;

export { Header, Link };
