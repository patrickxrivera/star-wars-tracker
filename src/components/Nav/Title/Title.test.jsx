import React from 'react';
import { mount } from 'enzyme';
import Title from './Title.jsx';

test('Title renders by default', () => {
  const wrapper = mount(<Title />);
  expect(wrapper).toMatchSnapshot();
});
