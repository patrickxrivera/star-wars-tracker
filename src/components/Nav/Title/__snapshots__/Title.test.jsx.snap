// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders by correctly 1`] = `
<Title>
  <div>
    <styled.div>
      <div
        className="sc-bdVaJa eEivRj"
      >
        The API Strikes Back
      </div>
    </styled.div>
  </div>
</Title>
`;
